docs/How-To-Create-Evergreen-Projects.md

```diff
@@ -0,0 +1,27 @@
+# How to Create Evergreen Projects
+
+This guide is meant to help users create their own projects on Evergreen without creating a ticket and waiting. 
+
+## Authorization
+
+To have access to create a project, the user must be either a project admins or super user.
```

Comment:
nit: project admin singular (although I would change this to "must be either a super user or the admin of an existing project"

Reply:
done